frontend/src/components/projectCreate/index.js

```diff
@@ -30,10 +30,80 @@ const ProjectCreationMap = React.lazy(() =>
   import('./projectCreationMap' /* webpackChunkName: "projectCreationMap" */),
 );
 
-var toGeojson = require('@mapbox/togeojson');
+var toGeojson = require('@tmcw/togeojson');
```

Comment:
With the new version, we can do the `import toGeojson from "@tmcw/togeojson";`

Reply:
Worked on it. I used `import { kml } from "@tmcw/togeojson";`